README.md

```diff
@@ -2,6 +2,8 @@
 
 LeaderWorkerSet: An API for deploying a group of pods as a unit of replication
 
+## Design
```

Comment:
At the least, let's mention the design proposal for reference. This can be helpful.